### DISCUSSION TITLE
Sunny Braille v1.0.0

### DISCUSSION THREAD

**Discussion by jun-brro** [sunnybraille/sunnybraille-client#58] [Announcements]
## What's Changed
* docs: PR 템플릿 추가 by @gitchannn in https://github.com/sunnybraille/sunnybraille-client/pull/3
* chore: client ci by @gitchannn in https://github.com/sunnybraille/sunnybraille-client/pull/5
* fix: npm dependency docz 제거 by @gitchannn in https://github.com/sunnybraille/sunnybraille-client/pull/2
* feat: Button, NavBar hover effect by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/6
* fix: Update cleint-ci.yml by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/7
* fix: Solved CI/CD Issues by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/8
* Create App.test.tsx by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/9
* Adding Jest.config.js by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/10
* Adding babel.config.js by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/11
* Update package.json by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/12
* feat: High Contrast mode by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/13
* Added fileName on UploadAlert by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/21
* ProgressStatus Update by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/22
* feat: UX improvement by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/23
* Fixing split error (temporary) by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/24
* added fileName variable (fixed eslint error) by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/25
* Feat: getting fileName from server when downloading by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/30
* feat: updated UX by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/31
* feat: Mobile UI sidebar by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/33
* fix: getting only content by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/34
* feat: updated UI by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/35
* feat: download aria live by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/46
* feat: Login session connection by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/47
* Feat: UX improvement by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/48
* feat: login, signup ui develop by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/49
* fix: api endpoint update by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/51
* Fix: api endpoint update (login, translate -> transcript) by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/52
* Fix: login session maintain by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/54
* Feat: Drag&Drop File Area by @JunhyeongPark-kr in https://github.com/sunnybraille/sunnybraille-client/pull/55

## New Contributors
* @gitchannn made their first contribution in https://github.com/sunnybraille/sunnybraille-client/pull/3
* @JunhyeongPark-kr made their first contribution in https://github.com/sunnybraille/sunnybraille-client/pull/6

**Full Changelog**: https://github.com/sunnybraille/sunnybraille-client/commits/1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/sunnybraille/sunnybraille-client/releases/tag/1.0.0'>Sunny Braille v1.0.0</a>.</em>

**Comment by gitchannn** [comment]
v1.0.0 배포 축하드립니다!
월요일에 파티하죠
